Src/quadratic.py

```diff
@@ -0,0 +1,22 @@
+# Solve the quadratic equation ax**2 + bx + c = 0
+import math
```

Comment:
```suggestion
from math import sqrt
```
Correct this then its good to merge

Reply:
As only sqrt function is used from the math module, that's why no need to import whole math module.